lib/server/socket.exs

```diff
@@ -0,0 +1,60 @@
+Code.require_file "../helpers/process_commands.exs", __DIR__
+
+defmodule Alchemist.Server.Socket do
+  use Application
```

Comment:
This looks very weird that it's an application.
As I see it doesn't need to be so.